src/main/java/org/cloudburstmc/api/event/inventory/CraftItemEvent.java

```diff
@@ -31,7 +31,7 @@ public CraftItemEvent(Player source, List<ItemStack> input, Recipe recipe) {
 
     public CraftItemEvent(Player player, ItemStack[] input, Recipe recipe) {
         this.player = player;
-        this.input = Lists.newArrayList(input);
+        this.input = new ArrayList<>();
```

Comment:
`new ArrayList<>(input)`?
Also I wonder, do we want input list to be mutable?
I feel like we don't, so `getInput` should return copy of input list.